index.js

```diff
@@ -86,12 +86,14 @@ function AutoIndex (db, idb, reduce) {
   }
 
   secondary.createValueStream = function (opts) {
-    (opts && opts || (opts = {})).keys = false
+    if (!opts) opts = {}
+    opts.keys = false
     return secondary.createReadStream(opts)
```

Comment:
@ungoldman can you double check my javascripting here?  This was some weird sort circuit shit that was mind bendy.